row_run.py

```diff
@@ -44,10 +45,27 @@ def mosaic_all_circles():
         row.format_time(perf_counter() - job_start),
     )
 
+
+def ocr_all_mosaics():
+    """the main function to execute when cloud run starts the ocr job"""
     job_start = perf_counter()
 
+    inputs = SimpleNamespace(
```

Comment:
praise: I haven't really done it before, but I like the idea of using SimpleNamespace here.

Reply:
yeah, just too many individual variables.